spec/models/organization_capacity_spec.rb

```diff
@@ -17,7 +17,7 @@
       context "for #{status} status" do
         let(:organization) { create :organization }
         before do
-          create :client_with_status, status: status[0], vita_partner: organization
+          create :client_with_status, status: status[0], vita_partner: organization, intake: create(:intake)
```

Comment:
i wonder if we should just add creating an intake to this factory since its pretty weird to have clients without an intake